README.md

```diff
@@ -1,11 +1,111 @@
-# ARES
-This repo has some code samples, documentation and usage of ARES test automation dashboard. This repo also has documentation on the usage, RESTAPI and other details
+<p><img src="https://i.ibb.co/kSgVFJF/ARES.png" width="300" height="150"></p>
 
-For more info on the tutorials, documentation, please refer to our wiki https://github.com/testastra/ARES/wiki
+# ARES DASHBOARD
 
-### Available Sample Code in 
+> ARES is a web based portal, which provides automation results at one place. It can integrate with most popular automation tools like Selenium, Protractor etc. This repo consist some samples and usage of ARES test automation dashboard
 
-* Java
-* Java Script
-* C#
-* Python (coming soon...)
+![HitCount](http://hits.dwyl.io/testastra/ares.svg)
+![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)
+![Open Source Love png1](https://badges.frapsoft.com/os/v1/open-source.png?v=103)
```

Comment:
It is not open source but under GNU license

Reply:
Removed open source badge